src/main/java/com/tictactoe/Program.java

```diff
@@ -10,12 +10,22 @@
     private GameSettings menu = new GameSettings(receiver, validator);
     private GamePreference preference = menu.collectGamePreference();
```

Comment:
`GamePreference` is a class that holds all the selected choices. It doesn't have any function.  Is that not good? or just the name is not good.